src/api/app/models/project.rb

```diff
@@ -109,7 +109,8 @@ def autocomplete(search)
   # will return all projects with attribute 'OBS:ImageTemplates'
   scope :local_image_templates, lambda {
     includes(:packages).joins(attribs: { attrib_type: :attrib_namespace }).
-      where(attrib_types: { name: 'ImageTemplates' }, attrib_namespaces: { name: 'OBS' })
+      where(attrib_types: { name: 'ImageTemplates' }, attrib_namespaces: { name: 'OBS' }).
+      order(:name)
```

Comment:
by ``:title``

and then you could drop the view because it is already sorted, right?